front/src/applications/operationalStudies/hooks/useScenarioData.ts

```diff
@@ -53,158 +60,194 @@ const useScenarioData = (scenario: ScenarioResponse, infra: InfraWithState) => {
       }
     );
 
-  const trainSchedulesResults = useMemo(
-    () => fetchedTrainSchedulesResults || [],
+  const { data: fetchedPacedTrains } =
+    osrdEditoastApi.endpoints.getAllTimetableByIdPacedTrains.useQuery(
+      { timetableId: scenario?.timetable_id },
+      {
+        skip: !scenario || !showPacedTrains,
+      }
+    );
+
+  const formattedRawTrainSchedules = useMemo(
+    () =>
+      (fetchedTrainSchedulesResults || []).map((trainSchedule) => ({
+        ...trainSchedule,
+        id: formatEditoastTrainIdToTrainScheduleId(trainSchedule.id),
+      })),
     [fetchedTrainSchedulesResults]
   );
 
-  const { trainScheduleSummariesById, setTrainScheduleSummariesById, allTrainsLoaded } =
-    useLazyLoadTrains({
+  const formattedRawPacedTrains: PacedTrainResultWithPacedTrainId[] = useMemo(
+    () =>
+      showPacedTrains
+        ? (fetchedPacedTrains || []).map((pacedTrain) => ({
+            ...pacedTrain,
+            id: formatEditoastTrainIdToPacedTrainId(pacedTrain.id),
+          }))
+        : [],
+    [showPacedTrains, fetchedPacedTrains]
+  );
+
+  // TODO Paced trains : pass setTimetableItemIdsToProject in https://github.com/OpenRailAssociation/osrd/issues/10613
+  const { timetableItemSummariesById, setTimetableItemSummariesById, allTimetableItemsLoaded } =
+    useLazyLoadTimetableItems({
       infraId: scenario.infra_id,
       electricalProfileSetId,
-      trainIdsToFetch,
-      trainSchedules,
-      setTrainIdsToProject,
+      timetableItemIdsToFetch,
+      timetableItems,
     });
 
   useEffect(() => {
-    if (allTrainsLoaded) {
-      setTrainIdsToFetch([]);
+    if (allTimetableItemsLoaded) {
+      setTimetableItemIdsToFetch([]);
     }
-  }, [allTrainsLoaded]);
+  }, [allTimetableItemsLoaded]);
 
+  // TODO Paced trains : remove this hook in https://github.com/OpenRailAssociation/osrd/issues/10791
   const { projectedTrainsById, allTrainsProjected, setProjectedTrainsById } = useLazyProjectTrains({
     infraId: scenario.infra_id,
     electricalProfileSetId,
     trainIdsToProject,
     path: projectionPath?.path,
-    trainSchedules,
-    moreTrainsToCome: !allTrainsLoaded,
+    timetableItems,
+    moreTrainsToCome: !allTimetableItemsLoaded,
     setTrainIdsToProject,
   });
 
+  // TODO Paced trains : create a new useLazyProjectTimetableItems in https://github.com/OpenRailAssociation/osrd/issues/10613
+
   useEffect(() => {
-    if (trainSchedules && projectionPath?.path && allTrainsLoaded) {
-      // TODO Paced train : Adapt this to handle paced trains in issue https://github.com/OpenRailAssociation/osrd/issues/10615
-      const trainIds = trainSchedules.map((trainSchedule) => trainSchedule.id);
+    if (timetableItems && projectionPath?.path && allTimetableItemsLoaded) {
+      // TODO Paced train : Adapt this to handle paced trains in issue https://github.com/OpenRailAssociation/osrd/issues/10613
+      const trainIds = timetableItems.map((timetableItem) => timetableItem.id);
```

Comment:
Hum should you take all the trains here only for the moment ? 
```suggestion
      const trainIds = timetableItems
        .filter((timetableItem) => isTrainSchedule(timetableItem.id))
        .map((timetableItem) => timetableItem.id);
```

Reply:
You're right. Done.